README.md

```diff
@@ -4,10 +4,27 @@ stackage
 [![Build Status](https://travis-ci.org/fpco/stackage.svg?branch=master)](https://travis-ci.org/fpco/stackage)
 
 "Stable Hackage": creating a vetted set of packages from Hackage.
+This repository is for package authors and maintainers to get their packages into Stackage.
+If you simply want to use Stackage as an end user, please follow the instructions on [http://www.stackage.org/](http://www.stackage.org).
```

Comment:
I would probably go more neutral, and just say "Stackage users go to stackage.org", and leave stackage.org to direct people as appropriate.